### ISSUE TITLE
Using date variables in templates

### ISSUE THREAD

**Issue by quartelmestre** [Balibaloo/obsidian-local-template-configuration#10]
**Is your feature request related to a problem? Please describe.**
outputs_to_templated_pathname does not accept date variables

**Describe the solution you'd like**
It would be helpful if outputs_to_templated_pathname could accept date variables such as {{date:YYYY}}, as per core plugins and several community plugins.

**Comment by Balibaloo** [comment]
Good suggestion,
For now as a quick and easy solution, I will add a format property to the natural date prompt.

**Comment by Balibaloo** [comment]
Closed by [2.2.2](https://github.com/Balibaloo/obsidian-local-template-configuration/releases/tag/2.2.2)

**Comment by quartelmestre** [comment]
Thanks, Roman! This patch helps indeed!

For the record, when using a `format`  for a `natural_date` variable, I had to write it without quotes:

```
    format: YYYY
```

Contrariwise, for prepopulating the field with today's date, I had to use quotes:

```
    is_initially: "@today"
```

Either using quotes for `format`, or foregoeing the quotes for `is_initially`, resulted in an error message stating that no intents were found. Nothing was output to the console.
